src/config/layers.js

```diff
@@ -795,6 +799,271 @@ export const grenzen = new Layer({
+      },
+    }),
+  ],
+});
+
+export const zweitausbildungPois = new Layer({
+  name: 'ch.sbb.zweitausbildung.tourist.pois.group',
+  visible: true,
+  properties: {
+    hasInfos: true,
+    layerInfoComponent: 'ZweitausbildungLayerInfo',
+    zweitausbildung: {
+      infos: {
+        title: 'ch.sbb.zweitausbildung.tourist.pois.group-title',
+      },
+    },
+  },
+  children: [
+    new ZweitausbildungPoisLayer({
+      name: 'ch.sbb.zweitausbildung.tourist.pois.no_railaway',
+      key: 'ch.sbb.zweitausbildung.tourist.pois.no_railaway',
+      visible: true,
+      zIndex: 4,
+      properties: {
+        popupComponent: 'ZweitausbildungPoisPopup',
+        hasInfos: true,
+        layerInfoComponent: 'ZweitausbildungSubLayerInfo',
+        zweitausbildung: {
+          viewparams: 'railway:false',
+          color: 'rgba(0, 61, 133, 0.8)',
+          infos: {
+            legend: [
+              {
+                image: 'poi_no_railaway.png',
+                name: 'ch.sbb.zweitausbildung.tourist.pois.no_railaway-name',
+              },
+            ],
+          },
+        },
+      },
+    }),
+    new ZweitausbildungPoisLayer({
+      name: 'ch.sbb.zweitausbildung.tourist.pois.railaway',
+      key: 'ch.sbb.zweitausbildung.tourist.pois.railaway',
+      visible: true,
+      zIndex: 4,
+      properties: {
+        popupComponent: 'ZweitausbildungPoisPopup',
+        hasInfos: true,
+        layerInfoComponent: 'ZweitausbildungSubLayerInfo',
+        zweitausbildung: {
+          viewparams: 'railaway:true',
+          color: 'rgba(235, 0, 0, 0.8)',
+          infos: {
+            legend: [
+              {
+                image: 'poi_railaway.png',
+                name: 'ch.sbb.zweitausbildung.tourist.pois.railaway-name',
+              },
+            ],
+          },
+        },
+      },
+    }),
+  ],
+});
+
+export const zweitausbildungRoutes = new Layer({
+  name: 'ch.sbb.zweitausbildung.linien.group',
+  visible: true,
+  properties: {
+    hasInfos: true,
+    layerInfoComponent: 'ZweitausbildungLayerInfo',
+    zweitausbildung: {
+      infos: {
+        title: 'ch.sbb.zweitausbildung.linien.group-title',
+      },
+    },
+  },
+  children: [
+    new ZweitausbildungRoutesLayer({
+      name: 'ch.sbb.zweitausbildung.tourist.routes.group',
+      key: 'ch.sbb.zweitausbildung.tourist.routes.group',
+      visible: false,
+      radioGroup: 'zweitausbildungRoutes',
+      properties: {
+        hasInfos: true,
+        layerInfoComponent: 'ZweitausbildungRoutesSubLayerInfo',
+        zweitausbildung: {
+          infos: {
+            title: 'ch.sbb.zweitausbildung.tourist.routes.group',
+            desc: 'ch.sbb.zweitausbildung.tourist.routes.group-desc',
+            legend: {
+              image: 'legend_tourist_strecken.png',
+            },
+          },
+          layer: 'zweitausbildung_tourist_strecken_grouped_qry',
+        },
+      },
+      children: [
+        new ZweitausbildungRoutesHighlightLayer({
```

Comment:
I have added the dropdown field as a child layer with the `layer.renderItemContent()` function. This dropdown should always be visible. I thought about adding an `expandWithParent` property to the layer. So, the LayerTree (react-spatial) always expands its child layers if one child has this property. @oterral what do you think?

Reply:
> I thought about adding an `expandWithParent` property to the layer. So, the LayerTree (react-spatial) always expands its child layers if one child has this property. 

I should better use an `expandChildren` property to the layer where its children should be expanded. Otherwise, it would be too complicated.

Nevertheless, what do you think about this react-spatial change, @oterral?

Reply:
yes a  boolean isAlwaysExpanded  that removes the expand functionnality and always display the children is a good idea.

Reply:
@gevos

Reply:
I will add the property which initially expands the children. 
@oterral Would you still show the expand arrow and offer the expand functionality?

Reply:
@oterral https://github.com/geops/react-spatial/pull/537